src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml

```diff
@@ -233,7 +233,7 @@
                             </fabric:FabricIconControl.Style>
                         </fabric:FabricIconControl>
                         <TextBlock x:Name="largeTextResult" KeyboardNavigation.IsTabStop="True" Focusable="True" FontWeight="SemiBold" FontSize="{DynamicResource ConstStandardTextSize}" Margin="8 0 0 0" 
-                               AutomationProperties.HelpText="{x:Static Properties:Resources.largeTextResultAutomationHelpText}" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=PrimaryFGBrush}">
+                               AutomationProperties.HelpText="{x:Static Properties:Resources.ColorContrast_LargeTextResultHelpText}" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=PrimaryFGBrush}">
```

Comment:
This was just a rename of the resource to be consistent with other resource names